test/e2e/specs/specs-playwright/wp-likes__post-spec.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import {
+	DataHelper,
+	LoginFlow,
+	NewPostFlow,
+	GutenbergEditorPage,
+	PostLikesComponent,
+	MyHomePage,
+	PublishedPostsListPage,
+} from '@automattic/calypso-e2e';
+
+/**
+ * Constants
+ */
+const quote =
+	'The foolish man seeks happiness in the distance. The wise grows it under his feet.\n— James Oppenheim';
+
+describe( DataHelper.createSuiteTitle( 'Likes (Post)' ), function () {
+	describe( 'Like a new post', function () {
+		let postLikesComponent;
+		let gutenbergEditorPage;
+
+		it( 'Log in', async function () {
+			const loginFlow = new LoginFlow( this.page, 'gutenbergSimpleSiteUser' );
+			await loginFlow.login();
+		} );
+
+		it( 'Start new post', async function () {
+			const newPostFlow = new NewPostFlow( this.page );
+			await newPostFlow.newPostFromNavbar();
+		} );
+
+		it( 'Enter post title', async function () {
+			gutenbergEditorPage = await GutenbergEditorPage.Expect( this.page );
+			const title = DataHelper.randomPhrase();
+			await gutenbergEditorPage.enterTitle( title );
+		} );
+
+		it( 'Enter post text', async function () {
+			await gutenbergEditorPage.enterText( quote );
+		} );
+
+		it( 'Publish and visit post', async function () {
+			await gutenbergEditorPage.publish( { visit: true } );
+		} );
+
+		it( 'Like post', async function () {
+			postLikesComponent = await PostLikesComponent.Expect( this.page );
+			await postLikesComponent.clickLikePost();
+		} );
+
+		it( 'Unlike post', async function () {
+			await postLikesComponent.clickLikePost();
```

Comment:
I think it would be better to separate the like & unlike helpers. Currently, it works as a toggle, which makes the `unlike` rely on a valid outcome of the `like` step. In theory, the `like` step could be actually unliking the post and the `unlike` step would be doing the opposite, and we wouldn't know that it happened 😄

Reply:
As mentioned in [this comment](https://github.com/Automattic/wp-calypso/pull/53291#discussion_r650145720) I have split up the methods into discrete actions.